libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/core/package-info.java

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
This copyright is different to [this](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/package-info.java). This is just an example, the one of the QnA follows the rest of copyrights.


I noticed that this way of Copyright is the one of the Samples, but in the SDKs is different